cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -147,13 +149,13 @@ function create_resource_from_string() {
   local -r delay=$3;
   local -r config_name=$4;
   local -r namespace=$5;
-  while [ ${tries} -gt 0 ]; do
-    echo "${config_string}" | ${KUBECTL} ${KUBECTL_OPTS} --namespace="${namespace}" apply -f - && \
+  while [ "${tries}" -gt 0 ]; do
+    echo "${config_string}" | ${KUBECTL} "${KUBECTL_OPTS}" --namespace="${namespace}" apply -f - && \
       log INFO "== Successfully started ${config_name} in namespace ${namespace} at $(date -Is)" && \
       return 0;
-    let tries=tries-1;
+    (( tries=tries-1 ));
```

Comment:
change reason:
```
In ./cluster/addons/addon-manager/kube-addons.sh line 154:
    let tries=tries-1;
    ^---------------^ SC2219: Instead of 'let expr', prefer (( expr )) .
```